themes/round-hp/assets/js/components/pokemon.js

```diff
@@ -0,0 +1,112 @@
+Vue.component("Pokemon", {
+  template: `
+    <div 
+      :class="{'pkmn': true, 'pkmn--fainted': isFainted, 'pkmn--empty': isEmptySlot}" 
+      :style="{'background': getBackgroundColor }"
+    >
+      <div v-if="typeof pokemon == 'object'" class="pkmn__wrapper">
+        <svg viewBox="0 0 36 36" class="pkmn__hp">
+          <path
+            d="M18 2.0845
+              a 15.9155 15.9155 0 0 1 0 31.831
+              a 15.9155 15.9155 0 0 1 0 -31.831"
+            fill="none"
+            :stroke="getHPColor"
+            stroke-width="1.5"
+            :stroke-dasharray="getHP + ', 100'"
+          />
```

Comment:
You're probably best using the circle SVG element? check simple-circles

Reply:
Actually, the advantage of creating a path is that it allows to use the dasharray as `${percentage}, 100`, which brings a bit more sanity to how the circle works in terms of flexibility. Since I'm not using the width and height property and use `viewBox` for the svg size, it also makes it more flexible/scalable/resizable, no matter what size I render the circle in, so it avoids me having to do a lot of crazy shenanigans. What I feel more inclined to do tho, is to used the TrimmedSprite component instead of just rendering the pokemon image

Reply:
Yeah that's fine! Just thought I'd make your life a bit easier when it comes to the resizing :D

Reply:
all good haha no worries, I've actually simplified it. Building the path is just a formula so it's quite simple :)

LMK if you'd like me to make any other changes or how you'd like to proceed with this ;)